pkg/decision/evaluator/condition_tree_test.go

```diff
@@ -426,6 +426,6 @@ func TestConditionTreeEvaluateAnAudienceTreeMultipleAudiences(t *testing.T) {
 		},
 		AudienceMap: audienceMap,
 	}
-	result = conditionTreeEvaluator.Evaluate(audienceTree, treeParams)
+	result, _ = conditionTreeEvaluator.Evaluate(audienceTree, treeParams)
```

Comment:
Do we need to add any test cases that involve the second return value from `conditionTreeEvaluator.Evaluate`?
@mikeng13 @msohailhussain